REFERENCE.md

```diff
@@ -74,8 +68,6 @@ Data type: `String[1]`
 
 The package name used by puppet resource
 
-Default value: $bird::params::package_name_v6
```

Comment:
This is still my issue with data in modules + puppet-strings: there's no way to say `Default value comes from Hiera`. It's why I still like the params.pp: it's explicit to the user.